### PR TITLE
fix(firehose): set correct handler event type

### PR DESCRIPTION
**Pull request by pevisscher**

### CODE REVIEW

**Comment by pevisscher** [comment]
Current dependencies on/for this PR:
* main
  * **PR #43** <a href="https://app.graphite.dev/github/pr/skyleague/event-horizon/43" target="_blank">Graphite</a>  👈
    * **PR #44** <a href="https://app.graphite.dev/github/pr/skyleague/event-horizon/44" target="_blank">Graphite</a> 
      * **PR #45** <a href="https://app.graphite.dev/github/pr/skyleague/event-horizon/45" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/skyleague/event-horizon/43?utm_source=stack-comment).

**Review by kmvisscher — APPROVED** [review]

**Comment by pevisscher** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/skyleague/event-horizon/43?utm_source=stack-comment).


Job Reference: o2hXZcUGA7x4CULUTBXA

**Pull request merged**
